docs/workloads/batch/configuration.md

```diff
@@ -23,6 +23,10 @@ predictor:
   env: <string: string>  # dictionary of environment variables
   log_level: <string>  # log level that can be "debug", "info", "warning" or "error" (default: "info")
   shm_size: <string>  # size of shared memory (/dev/shm) for sharing data between multiple processes, e.g. 64Mi or 1Gi (default: Null)
+  dependency_path: # (optional)
```

Comment:
nit: consider `dependency_paths` (although I don't have a strong preference either way).

Another option could be:

```yaml
dependencies:
  pip_path:
  conda_path:
  shell_path:
```

Reply:
maybe it doesn't even need the `_path` suffix

Reply:
I think I'd prefer `dependencies` too. Shorter is better. And `dependencies` will make the user think of the right thing. I think going without the `_path` suffix also makes sense. We'll have the documentation to back this up.